### PR TITLE
Feature logout

### PR DESCRIPTION
**Pull request by DiegoCuevas**
* Create logout function to the navbar
* Add logout button

### CODE REVIEW

File: App/src/components/Nabvar.js [hunk 1]
```diff
@@ -36,9 +43,7 @@ function Nabvar() {
           </Link>
         </li>
         <li css={{ marginLeft: "auto" }}>
-          <Link css={linkStyle} to="/login">
-            Logout
-          </Link>
+          <button onClick={handleLogoutClick}>Logout</button>
```
Comment (AngieCristina) [inline App/src/components/Nabvar.js:46]: What happened with the `   <Link css={linkStyle} to="/login">`

Comment (DiegoCuevas) [reply App/src/components/Nabvar.js:46]: I deleted that by the validation of the login, it was that if there is no user redirects it to the login so the link was unnecessary.

Comment (AngieCristina) [reply App/src/components/Nabvar.js:46]: Thank you for your kind response, have a good day : ) !

**Review by AngieCristina — DISMISSED** [review]

**Comment by MariethPerez** [comment]
I think, that you could consider to add some tests.

**Review by MariethPerez — APPROVED** [review]

**Review by AngieCristina — APPROVED** [review]

**Pull request merged**